src/main/java/Duke.java

```diff
@@ -1,10 +1,125 @@
+import  java.util.Scanner;
+
 public class Duke {
+
+    static final int MAX_NO_OF_TASKS = 100;
+
     public static void main(String[] args) {
+        showWelcomeMessage();
+
+        Task[] userTasks = new Task[MAX_NO_OF_TASKS];
+        int taskCounter = 0;
+        boolean exitLoopStatus = false;
+        Scanner inputCommand = new Scanner(System.in);
+        String userInput;
+
+        while(true){
+            userInput = inputCommand.nextLine();
+            String[] individualWords = userInput.split(" ", 2);
+            switch(individualWords[0].toLowerCase()){
+            case "list":
+                displayListOfActivities(userTasks, taskCounter);
+                break;
+            case "todo":
+                taskCounter = createTodoTask(userTasks, taskCounter, individualWords);
+                break;
+            case "deadline":
+                taskCounter = createDeadlineTask(userTasks, taskCounter, individualWords[1]);
+                break;
+            case "event":
+                taskCounter = createEventTask(userTasks, taskCounter, individualWords[1]);
+                break;
+            case "done":
+                markActivityAsDone(userTasks, individualWords);
+                break;
+            case "bye":
+                exitLoopStatus = terminateProgram();
+                break;
+            default:
+                System.out.println("Oops. Unknown command. Try again");
+            }
+            if (checkLoopStatus(exitLoopStatus)) {
```

Comment:
This is the correct if-conditional layout with the 'break' in a separate line, well done!